apis/types/extra.go

```diff
@@ -0,0 +1,52 @@
+// Code generated by go-swagger; DO NOT EDIT.
+
+package types
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	strfmt "github.com/go-openapi/strfmt"
+
+	"github.com/go-openapi/swag"
+)
+
+// Extra extra
+// swagger:model Extra
+type Extra struct {
```

Comment:
This file is not generated by  go-swagger with the current api definition.

Reply:
done.